src/test/java/io/vertx/core/http/HttpServerCloseHookTest.java

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright (c) 2011-2019 Contributors to the Eclipse Foundation
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0, or the Apache License, Version 2.0
+ * which is available at https://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR Apache-2.0
+ */
+
+package io.vertx.core.http;
+
+import io.vertx.core.AbstractVerticle;
+import io.vertx.core.Future;
+import io.vertx.core.buffer.Buffer;
+import io.vertx.core.net.PfxOptions;
+import io.vertx.test.core.VertxTestBase;
+import org.junit.Test;
+
+/**
+ * @author Thomas Segismont
+ */
+public class HttpServerCloseHookTest extends VertxTestBase {
+
+  @Test
+  public void deployHandlerShouldGetException() {
+    vertx.deployVerticle(new TestVerticle(), onFailure(throwable -> {
+      complete();
+    }));
+    await();
+  }
+
+  private static class TestVerticle extends AbstractVerticle {
+
+    @Override
+    public void start(Future<Void> startFuture) {
+      PfxOptions pfxOptions = new PfxOptions().setValue(Buffer.buffer("boom"));
```

Comment:
rename to _invalidOptions_ so it's clearer it's not correct (took some time to figure this it)

Reply:
I thought `boom` was clear enough :stuck_out_tongue_winking_eye: 

But right, I'll update and merge. Thanks for reviewing.